obsei/analyzer/ner_analyzer.py

```diff
@@ -35,7 +35,9 @@ def __init__(self, **data: Any):
                 self.tokenizer_name, use_fast=True
             )
         else:
-            tokenizer = None
+            tokenizer = tokenizer = AutoTokenizer.from_pretrained(
+                self.model_name_or_path, use_fast=True
+            )
```

Comment:
Let's simplify whole code section as follows -
```
tokenizer = AutoTokenizer.from_pretrained(
          self.tokenizer_name if self.tokenizer_name else self.model_name_or_path, use_fast=True
)
```

Reply:
Yes, this is much cleaner